components/homme/src/share/cxx/utilities/Hash.hpp

```diff
@@ -0,0 +1,69 @@
+/********************************************************************************
+ * HOMMEXX 1.0: Copyright of Sandia Corporation
+ * This software is released under the BSD license
+ * See the file 'COPYRIGHT' in the HOMMEXX/src/share/cxx directory
+ *******************************************************************************/
+
+#ifndef HASH_HPP
```

Comment:
Just to be safe, since "hash" is a common word, let's prepent HOMMEXX or something, so we don't accidentally risk to clash our header against God knows what package we may link to e3sm down the road...

Reply:
Good catch.